docs/examples.rst

```diff
@@ -336,6 +336,16 @@ Adding, editing and deleting comments is similarly straightforward::
     comment.update(body='updated comment body but no mail notification', notify=False)
     comment.delete()
 
+Get all images from a comment::
+
+    issue = jira.issue('JRA-1330')
+    regex_for_png = re.compile(r'\!(\S+?\.jpg)\|?\S*?\!')
```

Comment:
```suggestion
    regex_for_png = re.compile(r'\!(\S+?\.png)\|?\S*?\!')
```

? or maybe can change the variable name to jpg, whichever is fine with me

Reply:
validated on cloud with copy paste of a jpeg image ... I could add a few image types